src/modules/Elsa.Workflows.Management/Activities/WorkflowDefinitionActivity/WorkflowDefinitionActivity.cs

```diff
@@ -84,10 +84,10 @@ private async ValueTask OnChildCompletedAsync(ActivityCompletedContext context)
         await activityExecutionContext.CompleteActivityAsync(completeCompositeSignal?.Value);
     }
 
-    private void CopyInputOutputToVariables(ActivityExecutionContext context)
+    private async Task CopyInputOutputToVariables(ActivityExecutionContext context)
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.